src/main.rs

```diff
@@ -258,6 +260,13 @@ fn main() {
             }
         }
 
+        "clean" => {
+            match fs::remove_dir_all(&config.dest) {
```

Comment:
I'll definitely give it a shot and come back with any questions or issues I run into.